packages/backend-core/src/middleware/errorHandling.ts

```diff
@@ -9,9 +8,10 @@ export async function errorHandling(ctx: any, next: any) {
     const status = err.status || err.statusCode || 500
     ctx.status = status
 
-    if (status > 499 || env.ENABLE_4XX_HTTP_LOGGING) {
```

Comment:
Is the `ENABLE_4XX_HTTP_LOGGING` env var gone or have I just missed its usage now? If its gone, what does that mean for self hosters utilising it